### PR TITLE
do not output progressive validation loss for oaa with subsampling

### LINKED ISSUE #1344: --oaa_subsample reports after-training predictions
When --oaa_subsample option is enabled,  'average loss' reported by vw is no longer progressive validation loss, but is a training loss. This makes it impossible to rely on 'average loss' for model selection.

### PR DESCRIPTION
**Pull request by ataymano**
Since there is no way to output progressive validation loss for subsampling mode without increasing variance, 'N.A.' has to be reported always.

Fixes #1344

### CODE REVIEW

File: test/RunTests [hunk 1]
```diff
@@ -1706,3 +1706,8 @@ printf '3 |f a b c |e x y z\n2 |f a y c |e x\n' | {VW} --oaa 3 -q ef --audit
 {VW} -d train-sets/b1848_dsjson_parser_regression.txt --dsjson --cb_explore_adf -P 1
     train-sets/ref/b1848_dsjson_parser_regression.stderr
 
+# Test 190: one-against-all with subsampling
+{VW} -k --oaa 10 --oaa_subsample 5 -c --passes 10 -d train-sets/multiclass --holdout_off
+    train-sets/ref/oaa_subsample.stderr
+
+# Do not delete this line or the empty line above it
```
Comment (jackgerrits) [inline test/RunTests]: 👍
